examples/player/simid_player.html

```diff
@@ -23,15 +23,30 @@
     }
   </style>
   <script>
+    //if ad is non-linear allow user to insert specific dimensions
```

Comment:
I'm a fan of doing all player changes in one PR and all creative changes in a different PR.  Theoreticaly these two elements should function independently of one another.